.nix/bindlib.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, buildDunePackage }:
+
+# We need the very last version "bleeding edge" since previous versions don't use dune.
+
+buildDunePackage rec {
```

Comment:
Why do you need to put a specific Nix file for some OCaml opam package and not others ?

Reply:
Most of the dependencies are already within the nixpkgs depot as for instance [zarith](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/ocaml-modules/zarith/default.nix). However, neither Bindlib nor unionfind are present within nixpkgs, hence the additional files.